### PR TITLE
enabled preflighted CORS method 

### PR DESCRIPTION
**Pull request by damko**
I wasn't able to connect my [chrome extension](https://github.com/damko/nextcloud_bookmarks_chrome) to the API because of some CORS problems
By reading the [manual](https://docs.nextcloud.com/server/11/developer_manual/app/api.html) I found out that this line is required for CORS to work properly.

### CODE REVIEW

File: controller/rest/publiccontroller.php [hunk 1]
```diff
@@ -40,7 +41,7 @@ public function __construct($appName, IRequest $request, $userId, Bookmarks $boo
 	 * @NoAdminRequired
 	 * @NoCSRFRequired
 	 */
-	public function newBookmark($url = "", $item = array(), $title = "", $is_public = false, $description = "") {
```
Comment (blizzz) [inline controller/rest/publiccontroller.php]: I think this is a breaking change, also that one in line 76

File: controller/rest/publiccontroller.php [hunk 2]
```diff
@@ -93,7 +94,7 @@ public function newBookmark($url = "", $item = array(), $title = "", $is_public
 	 * @NoAdminRequired
 	 * @NoCSRFRequired
 	 */
-	public function editBookmark($record_id = null, $url = "", $item = array(), $title = "", $is_public = false, $description = "") {
+	public function editBookmark($record_id = null, $url = "", $tags = array(), $title = "", $is_public = false, $description = "") {
```
Comment (blizzz) [inline controller/rest/publiccontroller.php]: here a breaking change as wellthis and the change below makes the API

**Comment by blizzz** [comment]
Thank you for your contribution and sorry for the late reply! I think the parameter changes causes the tests to fail, also are not needed for the preflighted CORS support. Or am I wrong?

**Comment by marcelklehr** [comment]
Fixed by #338

**Pull request closed without merging**